### ISSUE TITLE
Threshold: Couldn't load SAPI5 synthesizer due to wave out not hookable

### ISSUE THREAD

**Issue by leonardder** [nvaccess/nvda#9963]
### Steps to reproduce:
This can also be reproduced using the gui, but using the python console is more straight forward.

1. Launch a Python console
2. Paste in the following

```
from synthDrivers.sapi5 import SynthDriver
SynthDriver()

```

### Actual behavior:
The following error in de log:

```
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "synthDrivers\sapi5.pyc", line 134, in __init__
  File "synthDrivers\sapi5.pyc", line 74, in ensureWaveOutHooks
  File "synthDrivers\sapi5.pyc", line 40, in __init__
RuntimeError: could not hook waveOutOpen
```

### Expected behavior:
WHen enabling sapi5 from the gui, speech works as normal.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
threshold-18146

#### Windows version:
Windows 10 1903

#### Name and version of other software in use when reproducing the issue:
Microsoft SAPI5 voices

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, a portable version of ALPHA works fine.

**Comment by leonardder** [comment]
Interesting enough, running threshold from source with exactly the same config loads sapi5 just fine.

**Comment by feerrenrut** [comment]
Fixed in commit 3c10f6d via PR #9964
